eng/pipelines/templates/steps/compilation.check.yml

```diff
@@ -1,39 +0,0 @@
-parameters:
-  - name: Artifact
```

Comment:
Is this no longer needed?

Reply:
Nope. It hasn't been used for a long, long time.